### PR TITLE
getFileStats return pointer

### PR DESCRIPTION
**Pull request by ranajahanzaib**
### Changes
-

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

**Review by peterlimg — CHANGES_REQUESTED** [review]
We can return `[]*sdk.FileStats` if that's what needed, the new `FilesDetails` is unnecessary as we know what the `remotePath` is.

File: wasmsdk/blobber.go [hunk 1]
```diff
@@ -46,8 +46,12 @@ func createDir(allocationID, remotePath string) error {
 	return allocationObj.CreateDir(remotePath)
 }
 
-// lisBlobbersForFile returns details about
-func getFileStats(allocationID, remotePath string) ([]sdk.FileStats, error) {
+type FilesDetails struct {
+	RemotePath string `json:"remote_path"`
+}
```
Comment (boddumanohar) [inline wasmsdk/blobber.go]: as this struct is no longer being used. can this be removed?

Comment (peterlimg) [reply wasmsdk/blobber.go]: @cnlangzi can you help to remove the unused `FilesDetails`

Comment (cnlangzi) [reply wasmsdk/blobber.go]: @peterlimg  done

**Review by boddumanohar — APPROVED** [review]
LGTM

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | TRUE  |
| system-tests  | master  |
| gosdk  | getFileStats-mods  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by cnlangzi — APPROVED** [review]
LGTM

**Comment by cnlangzi** [comment]
@peterlimg  please check it again. thanks

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**